### DISCUSSION TITLE
Wechselrichter geht 1- bis 2-mal täglich offline

### DISCUSSION THREAD

**Discussion by BlackFlag69** [tbnobody/OpenDTU#888] [Q&A]
Ich habe das nach openDTU-onBattery verschoben, weil ja dort die Unterstützung für Power Meter und Limiter her kommt.

**Comment by DPO99** [comment]
Hast Du das hier mal versucht? Einfach nur um zu sehen, ob die Hm auch starten, wenn die DTU quasi nackerd läuft?

Ich würde folgendes testen:
-backup ziehen
-dtu komplett neu flashen bzw auf Werksreset setzen
-wlan und Wechselrichter einrichten und sonst nix (kein mqtt und sonst was) machen und das Backup aber nicht einspielen (quasi frische dtu)
